types/gov.go

```diff
@@ -10,7 +10,6 @@ import (
 
 const (
 	ProposalStatusInvalid = "PROPOSAL_STATUS_INVALID"
```

Comment:
Can we replace all the usages of this custom value with the correct `govtypes.StatusXXX` value as well?

Reply:
there's no such status in the sdk so we created it here. It is used when the proposal is not found on chain and so we consider it deleted, also where we set `PROPOSAL_STATUS_INVALID`.